types/node/util.d.ts

```diff
@@ -51,7 +51,7 @@ declare module "util" {
     function isSymbol(object: any): object is symbol;
     /** @deprecated since v4.0.0 - use `value === undefined` instead. */
     function isUndefined(object: any): object is undefined;
-    function deprecate<T extends Function>(fn: T, message: string): T;
+    function deprecate<T extends Function>(fn: T, message: string, code?: string): T;
```

Comment:
This was added in v0.8.0 https://nodejs.org/api/util.html#util_util_deprecate_fn_msg_code

Can you please make changes in other existing supported versions too?
* 10.x https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aee73c479bd3f156b1468f3fd7e34811263f9e1a/types/node/v10/util.d.ts#L58
* 8.x https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aee73c479bd3f156b1468f3fd7e34811263f9e1a/types/node/v8/base.d.ts#L5661

Reply:
Done